subsys/bluetooth/mesh/brg_cfg_srv.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/mesh.h>
+
+#define LOG_LEVEL CONFIG_BT_MESH_MODEL_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(bt_mesh_brg_cfg_srv);
+
+const struct bt_mesh_model_op _bt_mesh_brg_cfg_srv_op[] = {
+	BT_MESH_MODEL_OP_END,
+};
+
+static int brg_cfg_srv_init(const struct bt_mesh_model *model)
+{
+	if (!bt_mesh_model_in_primary(model)) {
```

Comment:
Please take a look at my PR: https://github.com/zephyrproject-rtos/zephyr/pull/74583. Initialization should follow the similar pattern.